PWGHF/TableProducer/candidateSelectorLc.cxx

```diff
@@ -260,8 +264,11 @@ struct HfCandidateSelectorLc {
     return true;
   }
 
-  void process(aod::HfCand3Prong const& candidates,
-               TracksSel const&)
+  /// \brief process function w/o Bayes PID
```

Comment:
The description here is not correct. Maybe you could write something like "funtion to apply Lc selections, supporting normal and Bayes PID"